src/network/networkd-network.h

```diff
@@ -123,7 +122,6 @@ struct Network {
         bool dhcp_send_release;
         bool dhcp_send_decline;
         DHCPUseDomains dhcp_use_domains;
-        sd_ipv4acd *dhcp_acd;
```

Comment:
Ooops, that must have caused a lot of fun side-effects if anyone used the same .network file with two interfaces...